ecf/include/envir-p1.h

```diff
@@ -0,0 +1,41 @@
+# envir-p1.h
+export job=${job:-$PBS_JOBNAME}
+export jobid=${jobid:-$job.$PBS_JOBID}
+
+export RUN_ENVIR=emc
+export envir=%ENVIR%
+export MACHINE_SITE=%MACHINE_SITE%
+export SENDDBN=${SENDDBN:-%SENDDBN:YES%}
+export SENDDBN_NTC=${SENDDBN_NTC:-%SENDDBN_NTC:YES%}
+if [[ "$envir" == prod && "$SENDDBN" == YES ]]; then
+    export eval=%EVAL:NO%
+    if [ $eval == YES ]; then 
+      export SIPHONROOT=${UTILROOT}/para_dbn
+    else 
+      export SIPHONROOT=/lfs/h1/ops/prod/dbnet_siphon
+    fi
+    export SIPHONROOT=${UTILROOT}/fakedbn
+else
+    export SIPHONROOT=${UTILROOT}/fakedbn
+fi
```

Comment:
Based on the logic here, regardless of the conditions, the result is:
`export SIPHONROOT=${UTILROOT}/fakedbn`
Also, a variable `eval` is being exported for use somewhere down when `envir` == `prod` and `SENDDBN` is `YES`.
Is that the desired outcome?

Reply:
The developer will always use fakedbn to test. eval is ok here

Reply:
What does `eval` do?  Is it used anywhere other than the if-test?  
If the developer will always use fakeddbn, then the first if test `"$envir" == prod` is not necessary (and this whole block just boiled down to this):
```
export SIPHONROOT=${UTILROOT}/fakedbn
```